src/components/portal/portal.tsx

```diff
@@ -11,10 +14,11 @@ export class Portal extends React.PureComponent<PortalProps> {
 
     public render() {
         const {children, ...rest} = this.props;
+        const id = getRootId(this);
         this.portalContent = (
-            <div {...rest} data-automation-id="PORTAL">{children}</div>
+            <div id={id} {...rest} data-automation-id="PORTAL">{children}</div>
```

Comment:
lets not take the native `id` attribute. I discussed it with @tobich and we thought the best way to identify the portal would be with data-automation-id being the unique id.

Reply:
not sure if anyone would ever use this prop for the portal root but i will make this modification as its really test intended anyway so it makes more sense